src/context.tsx

```diff
@@ -43,4 +39,49 @@ const init = function(){
     if (storage) state.fallenHeroes= JSON.parse(storage)
 }
 init()
-console.log(state.fallenHeroes, state.restOfAliveHeroes)
\ No newline at end of file
+console.log(state.fallenHeroes, state.restOfAliveHeroes)
+
+*/
+
+export interface CampaignContextInterface {
+  campaign: Campaign;
+  setCampaign: Dispatch<SetStateAction<Campaign>>;
+}
+
+const defaultState = {
+  campaign: {
+    fallenHeroes: [],
+    restOfAliveHeroes: ["Aragorn", "Eowyn", "Beravor"],
+    allHeroes: ["Aragorn", "Eowyn", "Beravor"],
+  },
+  setCampaign: (campaign: Campaign) => {},
+} as CampaignContextInterface;
```

Comment:
Type casting should be avoided in favour of type annotation when possible because you could cast the type incorrectly and it may lead to errors. You can instead provide the type to the `createContext` function as a type argument.

Reply:
should be done, just double-check pls